database-plugins/src/main/java/io/cdap/plugin/db/common/DBBaseConfig.java

```diff
@@ -144,4 +144,9 @@ public String getReferenceName() {
   public String getColumnNameCase() {
     return columnNameCase;
   }
+
+  @Nullable
```

Comment:
instead of possibly returning null, would be better to check for null in the getter and return a boolean. Makes it less error-prone for the caller.

Reply:
done :)